src/pages/management/configuration-api/v3.5/index.mdx

```diff
@@ -3189,6 +3189,55 @@ curl -X POST \
 
 </Section>
 
+<Section>
+
+<Text>
+
+### Get Product Source
+
+This method retrieves the "source" parameters that were passed when activating the LiveChat product.
+
+#### Specifics
+
+|                 |                                                                            |
+| --------------- | -------------------------------------------------------------------------- |
+| HTTP Method     | **GET**                                                                    |
+| Method URL      | `https://api.livechatinc.com/v3.5/configuration/action/get_product_source` |
+| Required scopes | -                                                                          |
+
+#### Request
+
+This method accepts no parameters
+
+#### Response
+
+| Field         | Notes                                                                      |
+| ------------- | -------------------------------------------------------------------------- |
+| `source_id`   | The source ID parameter, as passed when activating the LiveChat product.   |
+| `source_type` | The source type parameter, as passed when activating the LiveChat product. |
+| `source_url`  | The source URL parameter, as passed when activating the LiveChat product.  |
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.5/configuration/action/get_product_source \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -H 'Content-Type: application/json' \
+```
+
+</CodeSample>
+
+<CodeResponse type="configuration" version="v3.5" json="getProductSource"/>
```

Comment:
The response field seems empty. Isn't it because the `payloads/management/v3.5/configuration-api/index.js` file has not been exported? 🤔

Reply:
Good catch, added the missing export 💪